config/config.nonprod.yaml

```
@@ -1,8 +1,8 @@
 ---
 # Action Config
-- whiteboard_tag: devtest
-  contact: tbd
-  description: DevTest whiteboard tag
+- whiteboard_tag: jst
+  contact: bsieber@mozilla.com
+  description: What does JST mean?
   enabled: true
   parameters:
     jira_project_key: JST
```

```suggestion
- whiteboard_tag: nonprodtest
  contact: bsieber@mozilla.com
  description: Nonprod testing whiteboard tag
  enabled: true
  parameters:
    jira_project_key: OSS
```

Perhaps we can ask for a new test project for jbi in both jira envs?

JST does not exist anymore and OSS is just a temporary project to place issues.